Tools/AP_Periph/Parameters.cpp

```diff
@@ -661,6 +681,13 @@ const AP_Param::Info AP_Periph_FW::var_info[] = {
     GSCALAR(temperature_msg_rate, "TEMP_MSG_RATE", 0),
 #endif
 
+    // @Param: OPTIONS
+    // @DisplayName: AP Periph Options
+    // @Description: Bitmask of AP Periph Options
+    // @Bitmask: 0:Disable continous sensor probe
+    // @User: Standard
+    GSCALAR(options, "OPTIONS", 1),
```

Comment:
for compatibility we would use the old define to set the default value

Reply:
Done